float32[] angles
float32 duration
bool constant_hz
---
bool data
